vignettes/simpleReporter.Rmd

```diff
@@ -29,10 +29,11 @@ The implementation should consist of 4 steps:
 
 1. Add modules user interface to the user interface of the app.
 2. Initialize Reporter instance.
-3. Create the Report Card function with two arguments: card and comment.
+3. Create the Report Card function with two optional arguments: card and comment.
 The function should build the Card step by step and assuming it is empty at the beginning,
 the optional comment argument is a string provided by the user when the card is added.
 If the comment argument is not specified then it is added automatically at the end of the Card.
+If the card argument is added then the `ReportCard` instance is automatically created for the user.
```

Comment:
```suggestion
If the card argument is added then the `ReportCard` instance is automatically created for the user otherwise the function should create the card itself.
```